File: Modul_2/Soal_3/PRAK203-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main(void) {
    float a, b, i, j, x, y;
    scanf("%f", &a);
    scanf("%f", &b);
    scanf("%f", &i);
    scanf("%f", &j);
    scanf("%f", &x);
    scanf("%f", &y);
    float hasil = (a-b) * i / j - (x + y);
    printf("%.3f", hasil);
}

File: Modul_2/Soal_3/PRAK203-2310817210015-MuhammadBukhariFitri.py
userInput = input().split()

# Test Case 1
if len(userInput) == 6:
    a, b, i, j, x, y = map(float, userInput)

# Test Case 2
elif len(userInput) == 2:
    a, b = map(float, userInput)
    i, j = map(float, input().split())
    x, y = map(float, input().split())

hasil = (a-b) * i / j - (x + y)
print(f"{hasil:.3f}")
